src/pharmpy/tools/common.py

```diff
@@ -144,6 +146,10 @@ def create_results(
             final_results = me.modelfit_results
             break
 
+    # Create links to input model and final model
+    context.store_key("input", ModelHash(input_model_entry.model))
+    context.store_key("final", ModelHash(best_model))
```

Comment:
Use `context.store_final_model_entry` and `context.store_input_model_entry` (or similar methods instead). (see other comment)